puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/builders/gemset_create.yaml

```diff
@@ -0,0 +1,4 @@
+- builder:
+    name: gemset_create
+    builders:
+      - shell: !include-raw scripts/gemset_create.sh
```

Comment:
I'm not sure how/where the gemset name would be generated, since currently it's taking part of the job name (I'm guessing the `cut` means there was a slash in the job name, but I can't remember for sure).

I guess a parameter could be written straight into the job YAML and passed into gemset_create?

Another thought I've had is to get rid of per-job+executor gemsets and leave gems installed permanently in a per-executor gemset.  This would mean we just use ${EXECUTOR_NUMBER} in the rvm plugin's gemset name, which ought to expand.